### DISCUSSION TITLE
Is laravel-medialibrary compatible with lighthouse/ grapghQL

### DISCUSSION THREAD

**Discussion by simbig** [spatie/laravel-medialibrary#2441] [Q&A] [answered]
Hi there,

I'd be interested in buying the pro version and was wondering if the react components/routes are usable with the custom graphql mutations from lighthouse (https://github.com/nuwave/lighthouse/blob/master/docs/master/digging-deeper/file-uploads.md)?
I only found information about the form requests..

**Comment by simbig** [comment, marked as answer]
@freekmurze could you let me know who is responsible for answering questions about the pro version?

**Reply by freekmurze** [reply]
We currently don't support GraphQL mutations.
